src/WebUI/Startup.cs

```diff
@@ -98,6 +103,11 @@ public void Configure(IApplicationBuilder app, IHostingEnvironment env, ILoggerF
                     name: "default",
                     template: "{controller=Home}/{action=Index}/{id?}");
             });
+
+            //Seed DataBase
+            await dataInitializer.InitializeDataAsync();
```

Comment:
Wouldn't this run every app restart? Why not just put seeding in seed method called on migration as always?